src/Psalm/Internal/Type/Comparator/AtomicTypeComparator.php

```diff
@@ -593,12 +593,10 @@ public static function isContainedBy(
                         return true;
                     }
 
-                    if ($codebase->methods->methodExists(
-                        new MethodIdentifier(
-                            $input_type_part->value,
-                            '__tostring'
-                        )
-                    )) {
+                    if ($codebase->methods->methodExists(new MethodIdentifier($input_type_part->value, '__tostring'))
```

Comment:
I feel like this would belong in another PR

Reply:
Ehh, maybe. I sort of considered it a minor fix to the extension's behavior, I added it because I removed the `__toString` from the stub since the function doesn't actually exist.

Reply:
ok, fine by me :)